### ISSUE TITLE
MIDI CC messages on channel 2 not working

### ISSUE THREAD

**Issue by HorstBaerbel** [vcync/modV#885]
**Operating system and application version**
 - OS: Fedora
 - Version 3.29.1

**Describe the bug**
When connecting a MIDI CC message not on channel #1 (Message 0xbN where N carries the channel) it can be connected (learn button works), but is not working afterwards.
My MIDI controller sends either (different "scene settings in controller"): 
Channel 1, CC 23, Value 127 (first message) + 0 (in second message)
or:
Channel 2, CC 18, Value 127 (first message, no second message) <-- Not working

Maybe related to #889

**To Reproduce**
1. Learn MIDI CC message where channel is not 1
3. Connection not working

**Expected behavior**
Connection should work after learning

**Comment by HorstBaerbel** [comment]
The second variant is a toggle switch btw: On first press it sends 127 to turn something on, then 0 on second press to turn off. I wanted to use this to toggle module enables. Should this be possible?

**Comment by 2xAA** [comment]
> The second variant is a toggle switch btw: On first press it sends 127 to turn something on, then 0 on second press to turn off. I wanted to use this to toggle module enables. Should this be possible?

Yes, this is the "latch" behaviour in the MIDI panel.

**Comment by HorstBaerbel** [comment]
> > The second variant is a toggle switch btw: On first press it sends 127 to turn something on, then 0 on second press to turn off. I wanted to use this to toggle module enables. Should this be possible?
> 
> Yes, this is the "latch" behaviour in the MIDI panel.

Per button? I guess it is only per controller, but I'll check it out, thanks.

**Comment by 2xAA** [comment]
Yes per controller. Most MIDI controllers have some sort of setup software
to change the behaviour of specific buttons though, so worth a look to see
if you can configure it through that. Definitely out of scope on modV’s
side for now.
